### PR TITLE
Install ign-plugin executable

### PR DESCRIPTION
**Pull request by scpeters**
Fix unstable build:

* [Build Status](https://build.osrfoundation.org/view/ign-garden/job/ign-plugin2-debbuilder/327/) https://build.osrfoundation.org/view/ign-garden/job/ign-plugin2-debbuilder/327/

testing with this branch

* [Build Status](https://build.osrfoundation.org/view/ign-garden/job/ign-plugin2-debbuilder/328/) https://build.osrfoundation.org/view/ign-garden/job/ign-plugin2-debbuilder/328/

### CODE REVIEW

**Review by chapulina — COMMENTED** [review]
@j-rivero , adding a new package for the CLI looks like a different approach from the one suggested for Transport in 

* https://github.com/ignition-release/ign-transport11-release/pull/4

Should we use the same pattern for all libraries?

File: ubuntu/debian/control [hunk 1]
```diff
@@ -36,12 +36,25 @@ Depends: libignition-plugin2 (= ${binary:Version}),
...
          ${misc:Depends}
 Multi-Arch: same
+Recommends: ignition-plugin-cli
 Description: Ignition Robotics Plugin Library - Development files
  Ignition Plugin is a component in the ignition framework, a set of libraries
  designed to rapidly develop robot applications.
  .
  Package contains the development files
 
+Package: ignition-plugin-cli
```
[the remark this answers is not in the document]
Comment (scpeters) [reply ubuntu/debian/control:46]: they should be installable side-by-side since the executable is going to `libexec/ignition/plugin2`

**Comment by j-rivero** [comment]
Testing it here [Build Status](https://build.osrfoundation.org/view/ign-garden/job/ign-plugin2-debbuilder/377/)

**Comment by scpeters** [comment]
> Testing it here [Build Status](https://build.osrfoundation.org/view/ign-garden/job/ign-plugin2-debbuilder/377/)

looks good to me

I would approve but I'm the author of this PR

**Comment by scpeters** [comment]
I'll just go ahead and merge

**Pull request merged**